doc-Managing_Infrastructure_and_Inventory/topics/Deleting_EBS_Volumes.adoc

```diff
@@ -1,11 +1,12 @@
-[[deleting_cinder_volumes]]
+[[deleting_ebs_volumes]]
 = Deleting a Volume
```

Comment:
@adahms Hi Andrew - Just wondering what the difference between this section (Deleting_EBS_Volumes.adoc) and the next "Deleting a Volume" section (Deleting_Volumes.adoc) are? Should this one be specifically about Amazon EBS volumes, like the filename/xref link? I'm just wondering if they are duplicate sections or this one needs EBS in it...